staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/conversion.go

```diff
@@ -352,12 +354,12 @@ func Convert_Slice_string_To_Slice_int32(in *[]string, out *[]int32, s conversio
 	return nil
 }
 
-// Convert_Slice_string_To_v1_DeletionPropagation allows converting a URL query parameter propagationPolicy
-func Convert_Slice_string_To_v1_DeletionPropagation(in *[]string, out *DeletionPropagation, s conversion.Scope) error {
+// Convert_Slice_string_To_Pointer_v1_DeletionPropagation allows converting a URL query parameter propagationPolicy
+func Convert_Slice_string_To_Pointer_v1_DeletionPropagation(in *[]string, out **DeletionPropagation, s conversion.Scope) error {
 	if len(*in) > 0 {
-		*out = DeletionPropagation((*in)[0])
+		**out = DeletionPropagation((*in)[0])
 	} else {
-		*out = ""
+		**out = DeletionPropagation("")
```

Comment:
Should it be empty DeletePropagation or nil?

Reply:
address it as `*out = nil`